src/components/common/Dialog/Dialog.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Dialog from 'material-ui/Dialog'
+import { MuiThemeProvider } from 'material-ui/styles'
```

Comment:
use Dialog from @materail-ui/core
MuiThemeProvider uses in _app.js
no need to duplicate it here